### PR TITLE
Examples syntax batch6

### PR DESCRIPTION
**Pull request by samdoran**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
- os_auth
- os_client_config
- os_image_facts
- os_networks_facts
- os_nova_flavor
- os_object
- os_server
- os_subnet_facts
- rax_files
- rax_files_objects
- mysql_db
- file
- uri
- cl_bond
- cl_bridge
- cl_img_install
- cl_interface
- cl_license
- cl_ports

##### ANSIBLE VERSION

```
ansible 2.2.0.0
  config file = /Users/sdoran/.ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY

Per Ansible Core team meeitng on [2016-11-03](https://meetbot.fedoraproject.org/ansible-meeting/2016-11-03/public_core_meeting.2016-11-03-15.02.html), change syntax in EXAMPLES in Core modules to use multi-line YAML.

### CODE REVIEW

File: network/basics/uri.py [hunk 1]
```diff
@@ -153,20 +153,24 @@
...
-- uri: url=http://www.example.com
+- name: Check that you can connect (GET) to a page and it returns a status 200
+  uri:
+    url: 'http://www.example.com'
 
 # Check that a page returns a status 200 and fail if the word AWESOME is not
 # in the page contents.
-- action: uri url=http://www.example.com return_content=yes
+- uri:
+    url: http://www.example.com
```
Comment (samdoran) [inline network/basics/uri.py:163]: Quotes are option for the `url:` parameter in the `uri` module.

Comment (samdoran) [reply network/basics/uri.py:163]: Not sure of the _exacet_ rule around this, but my testing shows it works with or without quotes and doesn't report a syntax error.

**Comment by gregdek** [comment]
Thanks @samdoran for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: network/basics/uri.py [hunk 2]
```diff
@@ -193,8 +197,8 @@
     return_content: yes
     HEADER_Cookie: "{{login.set_cookie}}"
 
-# Queue build of a project in Jenkins:
-- uri:
+- name: Queue build of a project in Jenkins:
```
Comment (gundalow) [inline network/basics/uri.py]: Delete trailing `:`

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

**Review by gundalow — APPROVED** [review]

**Pull request merged**